Library/Homebrew/cmd/tap.rb

```diff
@@ -15,28 +15,57 @@ def tap
 
   def install_tap user, repo, clone_target=nil
     tap = Tap.new user, repo
-    return false if tap.installed?
-    ohai "Tapping #{tap}"
-    remote = clone_target || "https://github.com/#{tap.user}/homebrew-#{tap.repo}"
-    args = %W[clone #{remote} #{tap.path}]
-    args << "--depth=1" unless ARGV.include?("--full")
-    safe_system "git", *args
-
-    formula_count = tap.formula_files.size
-    puts "Tapped #{formula_count} formula#{plural(formula_count, 'e')} (#{tap.path.abv})"
-
-    if !clone_target && tap.private?
-      puts <<-EOS.undent
+
+    if tap.installed?
+      ohai "#{user}/#{repo} already tapped. Changing priority."
```

Comment:
This should check tap priority and ensure that they vary before printing this message.